packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -746,25 +746,25 @@ export class MgtPerson extends MgtTemplatedComponent {
     switch (availability) {
       case 'Available':
         switch (activity) {
-          case 'Available':
-            presenceIcon = getSvg(SvgIcon.PresenceAvailable);
-            break;
           case 'OutOfOffice':
             presenceIcon = getSvg(SvgIcon.PresenceOofAvailable);
             break;
+          // OutOfOffice and Uknowns
+          default:
+            presenceIcon = getSvg(SvgIcon.PresenceAvailable);
+            break;
         }
         break;
       case 'Busy':
         switch (activity) {
-          case 'Busy':
-          case 'InACall':
-          case 'InAMeeting':
-            presenceIcon = getSvg(SvgIcon.PresenceBusy);
-            break;
           case 'OutOfOffice':
           case 'OnACall':
             presenceIcon = getSvg(SvgIcon.PresenceOofBusy);
             break;
+          // Busy,InACall,InAConferenceCall,InAMeeting, Unknown
+          default:
```

Comment:
```suggestion
          case 'Busy':
          case 'InACall':
          case 'InAMeeting':
          default:
```

Reply:
I'd prefer if we were explicit about which known items are also applied to the default case